pkg/rancher-components/src/components/Form/Radio/RadioButton.vue

```diff
@@ -67,25 +68,25 @@ export default {
     value(neu) {
       this.isChecked = this.val === neu;
       if (this.isChecked) {
-        this.$refs.custom.focus();
+        (this.$refs.custom as HTMLElement).focus();
       }
     }
   },
 
   methods: {
-    clicked(e) {
+    clicked(e: Event) {
```

Comment:
We can add directly the target and avoid to pass the whole event so we can avoid the `as` below.

Also we may return with a single condition:

```ts
    clicked(target?: HTMLElement) {
      if (this.isDisabled || target?.tagName === 'A') {
        return;
      }

      this.$emit('input', this.val);
    }
}